packages/snaps-controllers/src/snaps/registry/json.ts

```diff
@@ -0,0 +1,98 @@
+import { SnapsRegistryDatabase } from '@metamask/snaps-registry';
+import { SnapId } from '@metamask/snaps-utils';
+import { satisfiesVersionRange } from '@metamask/utils';
+
+import {
+  SnapRegistry,
+  SnapRegistryInfo,
+  SnapRegistryRequest,
+  SnapRegistryResult,
+  SnapRegistryStatus,
+} from './registry';
+
+// TODO: Replace with a Codefi URL
+const SNAP_REGISTRY_URL =
+  'https://cdn.jsdelivr.net/gh/MetaMask/snaps-registry@main/src/registry.json';
```

Comment:
Maybe this should be configurable as part of the constructor?

Reply:
Yeah, good point.